src/app/pages/parents/parents.component.html

```diff
@@ -1,2 +1,2 @@
-<yps-table-user [data]="parentsData" [columns]="columns"></yps-table-user>
+<yps-table-user [data]="service.ParentInfo" [columns]="columns"></yps-table-user>
```

Comment:
Why parent info

Reply:
because query has a lot of queries and it's named for GetParentInfo method

Reply:
It is a list of parents, am I right?